doc/docusaurus/docs/using-plinth/evaluating-plinth.md

```diff
@@ -0,0 +1,201 @@
+---
+sidebar_position: 12
+---
+
+# Evaluating CompiledCode for testing
```

Comment:
I'd just call it `Evaluating CompiledCode` or `Running CompiledCode`

Reply:
I agree that suffix "for testing" makes title less concise, but I felt like it serves an important purpose:
under the hood evaluator uses the `defaultCekParametersForTesting` and I wanted to highlight the fact that compiled code evaluation "for real" is likely going to produce different `ExBudget` values than "for testing".

Reply:
See above comment